static/index2.html

```diff
@@ -2,8 +2,8 @@
 <head>
     <title>Bank Connect</title>
     <link type="text/css" rel="stylesheet" href="style.css">
-    <link type="text/css" rel="preload" href="dropzone.css" as="style" onload="this.rel='stylesheet'">
-    <noscript><link rel="stylesheet" href="dropzone.css"></noscript>
```

Comment:
This control is highly dependent on JavaScript so putting noscript tag anywhere doens't make sense?